features/VER000_Versioning-and-revision-control.feature

```diff
@@ -0,0 +1,31 @@
+@implementer-agreement
+@VER
+@version1
+@E00020
+
+Feature: VER000 - Versioning and revision control
+    The rule verifies the capability to track changes to building data over time and maintain a comprehensive history of those changes.
+    https://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/concepts/Object_Attributes/Revision_Control/content.html
+
+
+    Scenario Outline: Check for activation
+
+        Given an IfcOwnerHistory
```

Comment:
```suggestion
        Given an IfcRoot
        Given its attribute OwnerHistory
```

I would start from IfcRoot. It's functionally a bit better because otherwise the OwnerHistory could still be somewhere floating around without actually being used in the model.